RFIDForIoT/CS/Mfrc522Lib/Mfrc522.cs

```diff
@@ -0,0 +1,305 @@
+﻿/*
+ This code is based on the library written by github user mlowijs.
+ The original library can be found here https://github.com/mlowijs/mfrc522-netmf
+ */
+using Mfrc522Lib.Constants;
+using System.Threading.Tasks;
+using Windows.Devices.Spi;
+using Windows.Devices.Gpio;
+using Windows.UI.Xaml.Controls;
+using System.Diagnostics;
+using System;
+using Windows.Devices.Enumeration;
+using System.Collections.Generic;
+
+namespace Mfrc522Lib
+{
+    public class Mfrc522
+    {
+        public SpiDevice _spi { get; private set; }
+        public GpioController IoController { get; private set; }
+        public GpioPin _resetPowerDown { get; private set; }
+        public int pincount;
+
+        private const string SPI_CONTROLLER_NAME = "SPI2";

```

Comment:
SPI_CONTROLLER_NAME isn't used anywhere.  Remove it?

Reply:
Available SPI devices gets recognize automatically in UPBOARD
SpiDevice.GetDeviceSelector();

Will remove this line